src/main/resources/templates/message-inspector.ftl

```diff
@@ -96,7 +96,7 @@
             <span class="label label-default">Compression:</span> ${msg.compressionCodec}
             <div>
             <a href="#" class="toggle-msg"><i class="fa fa-chevron-circle-right">&nbsp;</i></a>
-            <pre class="message-body">${msg.message!''}</pre>
+            <pre class="message-body"><xmp>${msg.message!''}</xmp></pre>
```

Comment:
From my understanding, the `<xmp>` tag is considered obsolete and deprecated.  This would probably be better handled at the Freemarker level as it would also address any XSS vulnerabilities related to displaying message data.

https://freemarker.apache.org/docs/dgui_misc_autoescaping.html